components/PublishUpdateBtnWithData.js

```diff
@@ -19,15 +22,86 @@ class PublishUpdateBtn extends React.Component {
   constructor(props) {
     super(props);
     this.onClick = this.onClick.bind(this);
+    this.state = {
+      updateNotificationAudience: 'Financial Contributors',
+    };
   }
 
   async onClick() {
     const { id } = this.props;
-    await this.props.publishUpdate(id);
+    const { updateNotificationAudience } = this.state;
+    await this.props.publishUpdate(id, updateNotificationAudience);
+  }
+
+  handleNotificationChange(selected) {
+    this.setState({ updateNotificationAudience: `${selected.value}` });
```

Comment:
No need for a template string here

```suggestion
    this.setState({ updateNotificationAudience: selected.value });
```